Tests/TuistKitTests/Services/FetchServiceTests.swift

```diff
@@ -192,4 +192,118 @@ final class FetchServiceTests: TuistUnitTestCase {
 
         XCTAssertFalse(dependenciesController.invokedUpdate)
     }
+
+    func test_fetch_path_is_found_in_root() async throws {
+        // Given
+        let stubbedPath = try temporaryPath()
```

Comment:
This is not a `stubbedPath`, it's an actual directory. Let's name it `temporaryDirectory`.